src/chain/member_info.rs

```diff
@@ -79,8 +79,14 @@ impl MemberInfo {
         self.age_counter.increment()
     }
 
+    // Increment the age.
+    #[allow(unused)]
+    pub fn increment_age(&mut self) {
+        self.age_counter = AgeCounter::from_age(self.age().saturating_add(1));
+    }
```

Comment:
It is unused because the code is commented out. This is related to `Treat infant differently for split etc` from #1835 Can just remove it and link it to an issue for that.

Reply:
Ah, OK then.

Reply:
Fixed in PR https://github.com/maidsafe/routing/pull/1951, with issue tracking what the dead code should do https://github.com/maidsafe/routing/issues/1953